src/phases/query/Multiplanner.yml

```diff
@@ -0,0 +1,194 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This file defines templates to use in multiplanner performance tests.
+
+QueryKnobTemplate:
+  Name: SetQueryKnobs
+  Type: RunCommand
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: {^Parameter: {Name: "active", Default: [0]}}
```

Comment:
For `default`, we could use incorrect values like a string. This way, we can ensure we do not miss a parameter when using this template. Something like https://github.com/mongodb/genny/blob/master/src/workloads/query/LimitSkip.yml#L34.
This comment applies to other places Parameter is used.